src/test/java/org/openrewrite/java/migrate/apache/commons/lang/ApacheCommonsStringUtilsTest.java

```diff
@@ -45,6 +45,20 @@ void defaultString() {
               class Foo {
                   String in = "foo";
                   String out = StringUtils.defaultString(in);
+                  boolean out = StringUtils.isEmpty(in);
+                  String[] out = StringUtils.split(in);
+                  String[] out = StringUtils.split(in, ", ");
+                  boolean out = StringUtils.equals(in, "string");
+                  String out = StringUtils.chop(in);
+                  String out = StringUtils.replace(in, "o", "z");
+                  String out = StringUtils.strip(in);
+                  String out = StringUtils.join(in);
+                  String out = StringUtils.deleteWhitespace(in);
+                  String out = StringUtils.abbreviate(in, 5);
+                  String out = StringUtils.trimToEmpty(in);
+                  String out = StringUtils.substringAfter(in, ",");
+                  String out = StringUtils.right(in, 5);
+                  String out = StringUtils.mid(in, 2, 4);
```

Comment:
This won't compile with the left side variables all named the same, right? Perhaps best to reassign per type, such that you have multiple statements `str = StringUtils.xyz(in, ...)` and `arr = StringUtils.split(in);`

Reply:
Hm ya, I can fix this. Its odd though because I do remember being able to run the test and it passed fine. I wonder why this wasn't a bigger issue?

Reply:
I think we don't validate that we can actually compile the output of the recipe, because that's not always the case when you migrate but need a different version of a dependency on the classpath suddenly. Hence why we need to make sure the code looks at least plausible, ideally compiles with a manual inspection.